src/types/hypixel.d.ts

```diff
@@ -203,6 +204,11 @@ export type SkyblockProfileMember = {
 			[slayer in SkyblockSlayerBoss]: {
```

Comment:
Would we rather use Record<SkyblockSlayerBoss, SkyblockSlayerInfo> as a convention? We've been using it throughout the rest of the typings